client/extensions/woocommerce/components/list/list-header/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+const ListHeader = ( { children } ) => {
```

Comment:
Please consider naming this component `ListTableHeader` instead to make it clearer that it belongs in the context of a table (because it emits a `thead`)